pacman/operations/rigged_algorithms/isomorph_check.py

```diff
@@ -0,0 +1,50 @@
+"""A short algorithm to check if there is an isomorphism of the placement of
+vertices by two separate placement algorithms. One of the algorithms must
+output to memory placements_copy and
+<param_type>MemoryPlacements2</param_type> in the algorithms_metadata.xml."""
+
+import os
+import logging
+
+
+class IsomorphicChecker(object):
+
```

Comment:
same thing about class saying what it does, call having doc string saying types.